src/Components/Bnglore.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
```

Comment:
Instead of banglore.js, it should be a City component, where you should pass city name etc as props.